test/bigtest/network/configs/budgets.js

```diff
@@ -1,9 +1,12 @@
 import { BUDGETS_API } from '../../../../src/common/const';
 
-// TODO: replace me with groups
 const configBudgets = server => {
-  server.get(BUDGETS_API, () => {
-    return [];
+  server.get(BUDGETS_API, (schema) => {
+    return schema.budgets.all();
+  });
+
+  server.get(`${BUDGETS_API}/:id`, (schema, request) => {
```

Comment:
`id` might be anything and `.attrs` might be called on undefined. Shouldn't we handle this here? 
cc @NikitaSedyx

Reply:
yep, it can be an error `can't get attrs from undefined`

Reply:
fixed